doc/classes/Window.xml

```diff
@@ -286,27 +323,34 @@
 			If [code]true[/code], the window will have no borders.
 		</member>
 		<member name="content_scale_aspect" type="int" setter="set_content_scale_aspect" getter="get_content_scale_aspect" enum="Window.ContentScaleAspect" default="0">
+			Specifies how the content's aspect behaves when the [Window] is resized. The base aspect is determined by [member content_scale_size].
 		</member>
 		<member name="content_scale_factor" type="float" setter="set_content_scale_factor" getter="get_content_scale_factor" default="1.0">
+			Specifies the base scale of [Window]'s content when its [member size] is equal to [member content_scale_size].
 		</member>
 		<member name="content_scale_mode" type="int" setter="set_content_scale_mode" getter="get_content_scale_mode" enum="Window.ContentScaleMode" default="0">
+			Specifies how the content is scaled when the [Window] is resized.
 		</member>
 		<member name="content_scale_size" type="Vector2i" setter="set_content_scale_size" getter="get_content_scale_size" default="Vector2i(0, 0)">
+			Base size of the content (i.e. nodes that are drawn inside the window). If non-zero, [Window]'s content will be scaled when the window is resized to a different size.
 		</member>
 		<member name="current_screen" type="int" setter="set_current_screen" getter="get_current_screen" default="0">
 			The screen the window is currently on.
 		</member>
 		<member name="exclusive" type="bool" setter="set_exclusive" getter="is_exclusive" default="false">
 		</member>
 		<member name="max_size" type="Vector2i" setter="set_max_size" getter="get_max_size" default="Vector2i(0, 0)">
+			If non-zero, the [Window] can't be resized to be bigger than this size.
 		</member>
 		<member name="min_size" type="Vector2i" setter="set_min_size" getter="get_min_size" default="Vector2i(0, 0)">
+			If non-zero, the [Window] can't be resized to be smaller than this size.
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="Window.Mode" default="0">
 			Set's the window's current mode.
 			[b]Note:[/b] Fullscreen mode is not exclusive fullscreen on Windows and Linux.
 		</member>
 		<member name="popup_window" type="bool" setter="set_flag" getter="get_flag" default="false">
+			If [code]true[/code], the [Window] will be considered a popup. Popups are sub-windows that blocks parent's input and don't show as separate windows in system's window manager's window list.
```

Comment:
It also auto closes when anything outside of it (except the `safe_rect` set by `DisplayServer`s `window_set_popup_safe_rect`method) is clicked, or it (or app) looses focus in any other way. While it's blocking input to the parent when opened, in reality it will block only the first event, since it will close the window.

Basically, it's only useful for menus and dropdowns like the one `OptionButton` use.

It might be better to rename it to something like `popup_menu` to be more specific.

Reply:
For some reason Window spawned at runtime can't be closed even with close button, so the auto-closing won't work either.

Reply:
> For some reason Window spawned at runtime can't be closed even with close button

In `Window` class, there's no default handler of `NOTIFICATION_WM_CLOSE_REQUEST` notification, and popup logic is sending this notification (same for the close button), `Popup` class and default dialogs should have it implemented.